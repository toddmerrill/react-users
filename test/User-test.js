import React from 'react';
import { User } from '../src/components/Users.js'
import ReactTestUtils from 'react-addons-test-utils';

const shallowRenderer = ReactTestUtils.createRenderer();

import mock from './user-mocks'

it('User renders correctly', () => {
    const tree = shallowRenderer.render(
        <User user={mock.user} onClick={mock.f} isCurrentUser={true}/>
    );
    expect(tree).toMatchSnapshot();
});
