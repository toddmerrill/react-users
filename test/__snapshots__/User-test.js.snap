// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`User renders correctly 1`] = `
<div
  className="listrow current-row"
  onClick={[Function]}
>
  <img
    src="test-file-stub"
  />
  test-first
   
  test-last
</div>
`;
